### ISSUE TITLE
Improve display of previous videos to participants

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#616]
# TL;DR
Make it easier for participants to navigate through and download their participation videos. 

# Narrative
As a participant, I may want to download my videos, show my child, or share with a friend. The current interface where you click to advance one at a time with a very small thumbnail display is hard to use:

![Screen Shot 2021-01-09 at 2 54 15 PM](https://user-images.githubusercontent.com/9554627/104107514-ac067180-528a-11eb-8ec7-941043be5d0d.png)

This interacts with recruitment since sharing videos is one ways parents could  help advertise.

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Participant can click to download a video, or information about how to download is clearly provided
- [ ] Videos are displayed big enough to see by default
- [ ] Navigation allows parent to go both forwards and backwards in the list, at minimum (possibly also jumping around)
